### PR TITLE
Replaced the 'datetime.utcnow()' method with 'datetime.now(timezone.utc)'

### LINKED ISSUE #209: Remove deprecated utcnow calls
`datetime.datetime.utcnow()` is deprecated. We should use `datetime.datime.now(datetime.timezone.UTC)` where we use utcnow today

### PR DESCRIPTION
**Pull request by guriWiz**
Resolved #209

### CODE REVIEW

**Comment by sigmavirus24** [comment]
@Guri-X thanks for the PR! It seems like You've missed a spot where we parse the timestamp from a previously recorded interaction. That parsing needs to be timezone aware for this change to pass tests

**Comment by Guri-X** [comment]
@sigmavirus24 Thanks for pointing out the issue. I will review the code again and will push the required changes.

**Comment by Guri-X** [comment]
@sigmavirus24 I fixed the issue. It should be able to clear all tests now.

File: src/betamax/util.py [hunk 1]
```diff
@@ -163,7 +163,7 @@ def add_urllib3_response(serialized, response, headers):
 
 
 def timestamp():
-    stamp = datetime.utcnow().isoformat()
+    stamp = datetime.now(timezone.utc).isoformat().split("+")[0]
```
Comment (sigmavirus24) [inline src/betamax/util.py:166]: Better to do 

```suggestion
    stamp = datetime.now(timezone.utc).strftime("%Y-%m-%dT%H:%M:%S")
```

**Review by sigmavirus24 — APPROVED** [review]

**Comment by ReenigneArcher** [comment]
Would it be possible to create a release with this included?

**Comment by ReenigneArcher** [comment]
I guess this project is dead. That's sad.

**Pull request merged**